src/utils/callbackSet.ts

```diff
@@ -0,0 +1,38 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+  export module Utils {
+    export class CallbackSet<CB extends Function> {
```

Comment:
Yes, exporting this class is the one way for us to use it.  However, it means that now this CallbackSet now falls under our public API.  So my main question is...  Are we ok with exposing this, since if we do, it's part of our public API.